.storybook/components/IconGrid/IconGrid.tsx

```diff
@@ -70,6 +70,5 @@ const IconGrid = () => (
   </div>
 );
 
-// TODO: not sure why this is failing the lint check when it has associated story file
```

Comment:
In the future can you provide links to tickets as well in case others want to "watch" tickets or add comments to them. I'm wondering if the error with this is the story file has a `.ts` extension but the component is `.tsx`?

Reply:
This is mitigated with https://github.com/chanzuckerberg/edu-design-system/pull/1456, where it just places the IconGrid story under Icon